.github/workflows/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+## PR Checklist
+<!-- Please check if your PR fulfills the following requirements: -->
+
+- [ ] The commit message follows our guidelines: [Code of conduct](https://github.com/patched-codes/patchwork/blob/main/CODE_OF_CONDUCT.md)
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+- [ ] Does this PR introduce a breaking change?
+- [ ] Include PR in release notes?
+
+
+## PR Type
+<!-- What kind of change does this PR introduce? -->
+<!-- Please check the one that applies to this PR using "x". -->
+
+- [ ] Bugfix
+- [ ] Feature
+- [ ] Code style update (formatting, local variables)
+- [ ] Refactoring
```

Comment:
Refactoring includes code style changes, lets remove the `Code style update (formatting, local variables)` checkbox.